cdap-ui/pom.xml

```diff
@@ -260,24 +260,6 @@
                   <arguments>run distribute</arguments>
                 </configuration>
               </execution>
-              <execution>
```

Comment:
wouldn't this prevent our build to run the unit test?

Reply:
Yes. Right now that unit test is seldom being updated or maintained. Since we now have separate build plans let's skip on running unit tests during build. If you feel the tests add value let me know.